tests/unit/s2n_client_auth_handshake_test.c

```diff
@@ -113,8 +113,8 @@ int s2n_test_client_auth_message_by_message(bool no_cert)
     EXPECT_NOT_NULL(server_config = s2n_config_new());
     EXPECT_NOT_NULL(client_config = s2n_config_new());
     EXPECT_SUCCESS(s2n_config_set_unsafe_for_testing(client_config));
-    EXPECT_SUCCESS(s2n_config_set_signature_preferences(server_config, "20200207"));
-    EXPECT_SUCCESS(s2n_config_set_signature_preferences(client_config, "20200207"));
+    EXPECT_SUCCESS(s2n_config_set_cipher_preferences(server_config, "default_tls13"));
+    EXPECT_SUCCESS(s2n_config_set_cipher_preferences(client_config, "default_tls13"));
```

Comment:
Does this test rely on the signature preference list including or not including something specific? If it does, we can't use the default, because the default can change.

Reply:
I went back and forth on this one, this test relies on `config->security_policy->signature_preferences` to be set to `s2n_signature_preferences_20200207`. The security_policy corresponding to this signature_preference is `security_policy_20190801`, I can add a new version  set called `20190801` to security_policy_selection[] to be more specific and change this to 

```
EXPECT_SUCCESS(s2n_config_set_cipher_preferences(server_config, "20190801"));
EXPECT_SUCCESS(s2n_config_set_cipher_preferences(client_config, "20190801"));
```